packages/xo-server/src/xo-mixins/backups-ng/index.js

```
@@ -551,6 +577,11 @@ export default class BackupNg {
         }
         await asyncMap(vms, handleVm)
       }
+
+      app.getLogger('restore').then(logger => {
```

What should we do if this call fails?

---

We should ignore errors and not log the tasks.

---

And show a warning in the console.

---

You can simply do like it's done in the `Api` mixin.